### DISCUSSION TITLE
Do we have Subscription or Observer Class or method in aws-java-sdk-appsync.

### DISCUSSION THREAD

**Discussion by Ujjwal9235** [aws/aws-sdk-java#2847] [Q&A]
Not enough documentation for Appsync in aws java sdk,
I need to implement Subscription or observer to Appsync api to push message to all the subscribers whenever there will be some message get pushed in Appsync api in JAVA. But i can't find any class or method in aws-java-sdk-appsync for subscription.

**Comment by debora-ito** [comment]
@Ujjwal9235 not sure I understand the question. [AWS AppSync](https://docs.aws.amazon.com/appsync/latest/devguide/what-is-appsync.html) is a service that provides GraphQL interface to databases.

Maybe you meant other service like AWS EventBridge? If not, can you elaborate on your use case?

**Reply by Ujjwal9235** [reply]
@debora-ito   I want to make a real-time java application with appsync in 100% Java, and for that I need java client which will able to subscribe and listen the message and print the message, whenever there will be any mutation.

**Reply by Ujjwal9235** [reply]
@debora-ito can u please help?
